aldryn_client/utils.py

```diff
@@ -155,6 +155,10 @@ def is_windows():
     return sys.platform == 'win32'
 
 
+def is_linux():
+    return sys.platform in ('linux', 'linux2')
```

Comment:
According to python docs, yes. But also according to python docs, one should use ``startswith``. Oh well.